src/jit/compiler.hpp

```diff
@@ -4280,6 +4280,42 @@ bool                Compiler::compIsForInlining()
     return (impInlineInfo != NULL);
 }
 
+//------------------------------------------------------------------------------
+// compIsUnixAmd64ABI : Returns true if the compiler is for compiling UnixAmd64 ABI.
+//
+// Arguments:
+//    None
+//
+// Return Value:
+//    True if the compiler is for compiling UnixAmd64ABI. False otherwise.
+inline
+bool Compiler::compIsUnixAmd64ABI()
+{
+#ifdef UNIX_AMD64_ABI
+    return true;
+#else // !UNIX_AMD64_ABI
+    return false;
+#endif // UNIX_AMD64_ABI
+}
+
+//------------------------------------------------------------------------------
+// compIsAmd64ABI : Returns true if the compiler is for compiling Amd64 ABI.
+//
+// Arguments:
+//    None
+//
+// Return Value:
+//    True if the compiler is for compiling Amd64ABI. False otherwise.
+inline
+bool Compiler::compIsAmd64ABI()
```

Comment:
I don't think this should be compIsAmd64ABI - it is more like compIsAmd64Target. Also, I think it would be better to define the (single) required predicate as something like "isMinimumArgAreaRequiredForCall" (someone else could probably come up with a better name).